docs/getting-started/iris_classification.md

```diff
@@ -0,0 +1,291 @@
+If the LIMINAL_HOME environemnet variable is not defined, home directory defaults to
+~/liminal_home directory.*
+
+### Liminal start
+The start command spins up 3 containers that load the Apache Airflow stack. Liminal's Airflow extension is responsible to execute the workflows defined in the liminal.yml file as standard Airflow DAGs.
+```BASH
+liminal start
+```
+
+It runs the following three containers: 
+* liminal-postgress
+* liminal-webserver
+* liminal-scheduler
+
+Once liminal server has completed starting up, you can navigate to admin UI in your browser:
+[http://localhost:8080](http://localhost:8080)
+
+
+![](../nstatic/iris-classification/airflow_main.png)
+
+***Important:** Set off/on toggle to activate your pipeline (DAG), nothing will happen otherwise!*
+
+You can go to graph view to see all the tasks configured in the liminal.yml file: 
+[http://localhost:8080/admin/airflow/graph?dag_id=my_datascience_pipeline](
+http://localhost:8080/admin/airflow/graph?dag_id=my_datascience_pipeline
+)
+
+#### Now lets see what actually happened to our task:
+![](../nstatic/iris-classification/airflow_view_dag.png)
+
+
+#### Click on “train” and you will get this popup:
+![](../nstatic/iris-classification/airflow_view_log.png)
+
+
+#### Click on “view log” button and you can see the log of the current task run:
+![](../nstatic/iris-classification/airflow_task_log.png)
+
+## Liminal YAML walkthrough
+* [Mounted volumes](#Mounted-volumes)
+* [Pipeline flow](#Pipeline-flow)
+
+### Mounted volumes
+Describe of the mounted volume in your liminal YAML:
+```YAML
+name: MyDataScienceApp
+owner: Bosco Albert Baracus
+volumes:
+  - volume: gettingstartedvol
+    claim_name: gettingstartedvol-pvc
+    local:
+      path: .
+```
+
+### Pipeline flow
+Setting up the pipeline flow with the following tasks:
+```YAML
+pipelines:
+  - pipeline: my_datascience_pipeline
+    ...
+    schedule: 0 * 1 * *
+    tasks:
+      - task: train
+        type: python
+        description: train model
+        image: myorg/mydatascienceapp
+        cmd: python -u training.py train
+        ...
+      - task: validate
+        type: python
+        description: validate model and deploy
+        image: myorg/mydatascienceapp
+        cmd: python -u training.py validate
+        ...
+```
+
+##### Each task will internally mount the volume defined above to an internal representation, described under the task section in the yml:
+
+```YAML
+pipelines:
+    ...
+    tasks:
+      - task: train
+        ...
+        env:
+          MOUNT_PATH: /mnt/gettingstartedvol
+        mounts:
+          - mount: mymount
+            volume: gettingstartedvol
+            path: /mnt/gettingstartedvol
+```
+###### We specify the `MOUNT_PATH` in which we store the trained model.
+
+## Evaluate the iris classification model
+
+Once the iris classification model is complete, you can launch a pod of the pre-built image which contains a flask server:
```

Comment:
Once the iris classification model is complete, you can launch a pod of the pre-built image which contains a flask server:
replace with 
Once the iris classification model training is completed and model is deployed (to the mounted volume), you can launch a pod of the pre-built image which contains a flask server: